selfdrive/athena/athenad.py

```diff
@@ -472,6 +475,32 @@ def log_handler(end_event):
       cloudlog.exception("athena.log_handler.exception")
 
 
+def stat_handler(end_event):
+  while not end_event.is_set():
+    last_scan = 0
+    try:
+      curr_scan = sec_since_boot()
+      if curr_scan - last_scan > 10:
+        for stat_entry in filter(lambda name: not name.startswith(tempfile.gettempprefix()), os.listdir(STATS_DIR)):
```

Comment:
```suggestion
        for stat_entry in (for fn in os.listdir(STATS_DIR) if not fn.startswith(tempfile.gettempprefix())):
```

The result is also not a `stat_entry` but a filename.

Reply:
personally like lambdas better, but up to you.